<launch>
  <rosparam command="load" file="$(find deploy_planner)/config/grid_map_landing_planner_filter_chain.yaml" />
  <!-- TODO: place yaml file here -->

  <node pkg="deploy_planner" type="deploy_planner_node" name="deploy_planner_node" output="screen">
    <rosparam command="load" file="$(find deploy_planner)/config/grid_map_landing_planner_filter_chain.yaml" />
  	<param name="probe_range_limit_x" value="10.0" />
  	<param name="probe_range_limit_y" value="10.0" />
  	<param name="probe_range_limit_z_down" value="3.0" />
  	<param name="probe_range_limit_z_up" value="1.0" />
  	<param name="visualize_position" value="true" />
  	<param name="visualize_grid_map" value="true" />
  	<param name="traversability_threshold" value="0.8" />
  </node>

  <!-- Launch the grid map visualizer -->
  <node pkg="grid_map_visualization" type="grid_map_visualization" name="grid_map_visualization" output="screen" />

</launch>
